esp/bindings/http/platform/httpbinding.cpp

```diff
@@ -260,6 +259,68 @@ EspHttpBinding::EspHttpBinding(IPropertyTree* tree, const char *bindname, const
     }
     if(m_challenge_realm.length() == 0)
         m_challenge_realm.append("ESP");
+
+    if(!m_secmgr.get())
+        return;
+
+    processName.set(procname);
+    const char* authDomain = bnd_cfg->queryProp("@authDomain");
+    if (!isEmptyString(authDomain))
+        domainName.set(authDomain);
+    else
+        domainName.set("default");
+    domainSessionSDSPath.setf("%s/%s[@name=\"%s\"]/%s[@name=\"%s\"]/", PathSessionRoot, PathSessionProcess,
+        procname, PathSessionDomain, domainName.str());
+
+    domainAuthType = AuthPerRequestOnly;
+    VStringBuffer xpath("AuthDomains/AuthDomain[@domainName=\"%s\"]", domainName.get());
+    IPropertyTree* authDomainTree = proc_cfg->queryPropTree(xpath);
+    if (!authDomainTree)
+        throw MakeStringException(-1, "Can't find AuthDomain %s", domainName.str());
+
+    const char* authType = authDomainTree->queryProp("@authType");
+    if (isEmptyString(authType) || strieq(authType, "AuthTypeMixed"))
+        domainAuthType = AuthTypeMixed;
+    else if (strieq(authType, "AuthPerSessionOnly"))
+        domainAuthType = AuthPerSessionOnly;
+    if (domainAuthType == AuthPerRequestOnly)
+        return;
+
+    unsigned sessionTimeoutMinutes = authDomainTree->getPropInt("@sessionTimeoutMinutes", 0);
+    sessionTimeoutSeconds = sessionTimeoutMinutes < 0 ? -1 : sessionTimeoutMinutes > 0 ? sessionTimeoutMinutes * 60 : ESP_SESSION_TIMEOUT;
+
+    //The @resourceURL contains URLs which may be used before a user is authenticated.
+    //For example, an icon file on the login page.
+    const char* resourceURLs = authDomainTree->queryProp("@resourceURL");
+    if (isEmptyString(resourceURLs))
+        return;
+
+    StringArray urlArray;
+    urlArray.appendListUniq(resourceURLs, ",");
+    ForEachItemIn(i, urlArray)
+    {
+        const char* url = urlArray.item(i);
+        if (isEmptyString(url))
+            continue;
+        const char* star = strchr(url, '*');
```

Comment:
If you are using jlib regex WildMatch routines, there is also a function "isWildString()" you can use to check.  It checks for both * and ?.